spec/controllers/state_file/questions/nj_retirement_warning_controller_spec.rb

```diff
@@ -31,13 +56,15 @@
         it "does NOT show" do
           allow_any_instance_of(Efile::Nj::NjRetirementIncomeHelper).to receive(:show_retirement_income_warning?).and_return(false)
           expect(described_class.show?(intake)).to eq false
+          expect(intake.eligibility_retirement_warning_continue).to eq("unfilled")
```

Comment:
[pebble] since `show?` no longer updates this column, i don't think there's a point in testing the value here